extensions/kafka-client/deployment/src/main/java/io/quarkus/kafka/client/deployment/DevServicesKafkaProcessor.java

```diff
@@ -35,6 +39,8 @@
     private static final Logger log = Logger.getLogger(DevServicesKafkaProcessor.class);
     private static final int KAFKA_PORT = 9092;
     private static final String KAFKA_BOOTSTRAP_SERVERS = "kafka.bootstrap.servers";
+    private static final String DEV_SERVICE_LABEL = "quarkus-dev-service";
```

Comment:
I wonder if this should be configurable, in case you want some isolated clusters.

Reply:
We can either make the label configurable or the value. 
Right now, it's "kafka", but we can allow the user to configure it to something like "my-own-kafka-broker".

Reply:
(I've added a todo)

Reply:
Fixed.